packages/cw-utils/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "cw-utils"
+version = "2.0.0-rc.0"
+authors = ["Ethan Frey <ethanfrey@users.noreply.github.com>"]
+edition = "2021"
+description = "Common helpers for other cw specs"
+license = "Apache-2.0"
+repository = "https://github.com/CosmWasm/cw-minus"
+homepage = "https://cosmwasm.com"
```

Comment:
Could you remove all those `homepage` entries? They are not really helpful because not package specific and kindof pointless to have here: https://crates.io/crates/cw-utils

Reply:
No problem